cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -56,7 +61,30 @@ function LogViewerController ($scope, LogViewerStore, myLogsApi) {
     theColumns.push('time');
   }
 
-  var collapseCount = 0;
+  angular.forEach($scope.displayOptions, (value, key) => {
+    this.configOptions[key] = value;
+  });
+
+  this.logEvents = ['ERROR', 'WARN', 'INFO', 'DEBUG', 'TRACE'];
+
+  let included = {
+    'ERROR' : false,
+    'WARN' : false,
+    'INFO' : false,
+    'DEBUG' : false,
+    'TRACE' : false
```

Comment:
The default behavior is that all logs are shown, while none of the checkboxes are selected. When the user chooses to filter based on specific event types, they can elect to 'include' whichever event type they'd like in the filtering. By default, there are no event types included in the filtering - which is why their values are defaulted to false. 

The primary purpose is to narrow down the entries to those the user is looking for - given this behavior, to view a specific event type requires a single click, whereas the inverse would require 4 clicks to unselect the others to view a single event type. 

If a user wants to view 4 event types, it requires 4 clicks. If they want to view a single event, it requires a single click. This seems much more intuitive than the inverse.